spec/writers/datacite_writer_spec.rb

```diff
@@ -169,7 +169,7 @@
       subject.type = "Dataset"
       expect(subject.doi).to eq("123")
       expect(subject.valid?).to be false
-      expect(subject.errors.first).to start_with("Element '{http://datacite.org/schema/kernel-4}identifier'")
+      expect(subject.errors.first).to start_with("3:0: ERROR: Element '{http://datacite.org/schema/kernel-4}identifier'")
```

Comment:
Do you mean 3:0 ? The other ones say 4:0

Reply:
Yes. This is the schema version, and different examples might use different versions.